Kubernetes/linux/addons/kube-dns-depolyment.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
```

Comment:
Filename has a typo "depolyment"

Reply:
Yeah, I had fixed that at some point, then reverted back because I was scrambling for things to blame for not working. :)